unified_planning/walkers/type_checker.py

```diff
@@ -233,7 +238,7 @@ def walk_div(self, expression, args):
     @walkers.handles(OperatorKind.LE, OperatorKind.LT)
     def walk_math_relation(self, expression, args):
         for x in args:
-            if x is None or not (x.is_int_type() or x.is_real_type()):
+            if x is None or not (x.is_int_type() or x.is_real_type() or x.is_time_type()):
```

Comment:
Not sure if needed, but this only allows for `<` and `<=` relations. The equality between a time and a real constant is forbidden (see line 256/261 of this file) (that would be equivalent to the conjunction of two `<=` constraints BTW)

Reply:
Yes, currently the API does not allow constructing so I did not implement them as they would not have been tested. It is definitly something that should indeed be supported in the future :+1:

Reply:
OK! Can you add a `#TODO` or an issue to keep track of this?